torch/csrc/autograd/profiler.cpp

```diff
@@ -37,13 +39,19 @@ void mark(std::string name, bool include_cuda /* = true */) {
   }
 }
 
-void pushRange(std::string name) {
+void pushRange(std::string name, std::string msg/*= {}*/, int64_t sequence_nr/*= -1*/) {
```

Comment:
Can you please make `msg` a `const char *`?

Reply:
Yeah, our people need their NVTX...

I'll push this later today.  I will also think about further optimizations (for example, there's no reason nested ranges in the forward pass need to append the sequence number except at the top level).

Thanks!